src/main/kotlin/org/rust/lang/core/resolve2/FacadeResolve.kt

```diff
@@ -100,19 +100,18 @@ fun processMacros(
     processor: RsResolveProcessor,
     /** Needed to filter textual scoped macros if path is unqualified. */
     macroPath: RsPath,
-    isAttrOrDerive: Boolean,
 ): Boolean {
     val info = getModInfo(scope) ?: return false
-    return info.modData.processMacros(macroPath, isAttrOrDerive, processor, info)
+    return info.modData.processMacros(macroPath, processor, info)
 }
 
 private fun ModData.processMacros(
     macroPath: RsPath,
-    isAttrOrDerive: Boolean,
     processor: RsResolveProcessor,
     info: RsModInfo,
 ): Boolean {
     val isQualified = macroPath.qualifier != null
+    val isAttrOrDerive = macroPath.parent.let { it is RsMetaItem || it is RsUseSpeck }
```

Comment:
Hmm, why `|| it is RsUseSpeck`? I can't find a line where previously we passed `isAttrOrDerive = true` in the case of `RsUseSpeck`